docs/install.md

```diff
@@ -73,13 +73,13 @@ noverify version
 
 You're done! Proceed to the [Getting started](/docs/getting_started.md) page.
 
-## With `go get`
+## With `go install`
```

Comment:
these changes should be in a separate PR as they do not relate to its main topic

Reply:
I am totally agree with you (and that was the original title of this PR), but we had an agreement with @Danil42Russia  that this PR would also include changes to the documentation, corrections to some warnings, and test data. But if you insist, I can separate them

Reply:
ok!